src/codec_aom.c

```diff
@@ -43,6 +43,13 @@
 #endif
 #endif
 
+// Speeds 7-9 were added to all intra mode in https://aomedia-review.googlesource.com/c/aom/+/140624.
+#if defined(AOM_EXT_PART_ABI_VERSION)
+#if AOM_ENCODER_ABI_VERSION >= (10 + AOM_CODEC_ABI_VERSION + AOM_EXT_PART_ABI_VERSION)
+#define ALL_INTRA_HAS_SPEEDS_7_TO_9 1
+#endif
+#endif
```

Comment:
Please move the `#endif` at line 44 after this line, so that lines 46-51 are within the `#if defined(AVIF_CODEC_AOM_ENCODE)` block that starts on line 38.

The reason is that Clang has a warning about a macro that is defined but not used. Since the ALL_INTRA_HAS_SPEEDS_7_TO_9 macro is only used when AVIF_CODEC_AOM_ENCODE is defined, we should define this macro only under the same condition.

Reply:
Done